src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs

```diff
@@ -431,6 +434,163 @@ public void TestSerializeDictionaryStringTNoComparerNull()
             Assert.Equal(value, deserializedValue);
         }
 
+        [Theory]
+        [InlineData("en")]
+        [InlineData("en-US")]
+        [InlineData("en-CA")]
+        [InlineData("zh-HK")]
+        [InlineData("sr-Cyrl-CS")]
+        public void CultureInfo(string name)
+        {
+            CultureInfo value = new CultureInfo(name);
+            TranslationHelpers.GetWriteTranslator().Translate(ref value);
+
+            CultureInfo deserializedValue = null;
+            TranslationHelpers.GetReadTranslator().Translate(ref deserializedValue);
+
+            Assert.Equal(value, deserializedValue);
+        }
+
+        [Fact]
+        public void CultureInfoAsNull()
+        {
+            CultureInfo value = null;
+            TranslationHelpers.GetWriteTranslator().Translate(ref value);
+
+            CultureInfo deserializedValue = null;
+            TranslationHelpers.GetReadTranslator().Translate(ref deserializedValue);
+
+            Assert.Null(deserializedValue);
```

Comment:
When I was choosing assertion library, based on https://blog.safnet.com/archive/2018/12/03/assertion-comparisons/, i liked FluentAssertion better. Mostly for documentation part. Situation in 2021 could be different and I understand that there is some inertia with using Shouldly and I don't think we will or shall change it. Just saying.
So is there consensus in team to use Shouldly from now on? I have seen it used in few places but was not sure about the strategy here, so I have decided to fall back to g'old xunit asserts, although, personally, I like to have assertion library isolated from unit test runner and I love fluent assertions regardless it is shouldly or FluentAssetions.

Reply:
That, at least, was the previous consensus. We can bring it up again if you'd like—I don't care too much about documentation because once I've figured something out, I can just copy it rather than looking at docs again. I also appreciate fewer characters in the code.

Reply:
done